basics/data-import/pinot-file-system/amazon-s3.md

```diff
@@ -1,6 +1,11 @@
+---
+description: >-
+  This guide shows you how to import data from files stored in Amazon S3
+---
+
 # Amazon S3
 
-You can enable [Amazon S3](https://aws.amazon.com/s3/) Filesystem backend by including the plugin `pinot-s3` .
+Enable the [Amazon S3](https://aws.amazon.com/s3/) Filesystem backend by including the `pinot-s3` plugin. In the controller or server, add the config:
```

Comment:
```suggestion
Enable the [Amazon S3](https://aws.amazon.com/s3/) file system backend by including the `pinot-s3` plugin. In the controller or server, add the config:
```

Reply:
is this on the controller or server, or in the controller or server configuration?

Reply:
I'm going to submit a new PR with the additional changes, since this was already merged.